### PR TITLE
'm_angularOnly' is not initialized.

### PR DESCRIPTION
**Pull request by dotquixote**

### CODE REVIEW

File: servers/physics/joints/cone_twist_joint_sw.cpp [hunk 1]
```diff
@@ -107,6 +107,7 @@ ConeTwistJointSW::ConeTwistJointSW(BodySW *rbA, BodySW *rbB, const Transform &rb
 	B->add_constraint(this, 1);
 
 	m_appliedImpulse = 0;
+	m_angularOnly = 0;
```
Comment (akien-mga) [inline servers/physics/joints/cone_twist_joint_sw.cpp]: That works, but `false` would be better. Maybe also moved up next to `m_solveTwistLimit` to match the header: https://github.com/dotquixote/godot/blob/051146decf897b55fabc894ede8371d5cb86c538/servers/physics/joints/cone_twist_joint_sw.h#L100

**Comment by akien-mga** [comment]
See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase regarding how to edit your commit.

**Comment by dotquixote** [comment]
thanks! akien-mga

**Comment by akien-mga** [comment]
Merged as 23f10c31.

**Pull request closed without merging**